index.js

```diff
@@ -27,6 +27,16 @@ module.exports = function (opts) {
 		throw err;
 	}
 
+	// run autofix over as many errors as possible
+	if (opts.fix) {
+		overrides.maxErrors = Infinity;
+	}
+
+	checker.getConfiguration().override(overrides);
```

Comment:
But that's already handled by the above `if` statement. It will only override when `opts.fix`.